.scripts/prerequisites.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env sh
+
+if ! which xcodebuild >/dev/null; then
+  echo "xcodebuild does not seem to be installed. You can install it by installing Xcode from the App Store."
+  exit 1
+fi
+
+if ! which pod >/dev/null; then
+  echo "Cocoapods does not seem to be installed. You may need to run 'gem install cocoapods'."
+  exit 1
+fi
+
+if ! which xcpretty >/dev/null; then
```

Comment:
These tools are all mostly dead :(
The project seems mostly dead: https://github.com/xcpretty/xcpretty/issues/360